internal/process/provisioning/create_runtime_resource_step.go

```diff
@@ -81,9 +83,16 @@ func (s *CreateRuntimeResourceStep) Run(operation internal.Operation, log logrus
 	runtimeResourceName := steps.KymaRuntimeResourceName(operation)
 	log.Infof("KymaResourceName: %s, KymaResourceNamespace: %s, RuntimeResourceName: %s", kymaResourceName, kymaResourceNamespace, runtimeResourceName)
 
+	values, err := provider.GetPlanSpecificValues(&operation, s.config.MultiZoneCluster, s.config.DefaultTrialProvider, s.useSmallerMachineTypes, s.trialPlatformRegionMapping, s.config.DefaultGardenerShootPurpose)
+	if err != nil {
+		return s.operationManager.OperationFailed(operation, fmt.Sprintf("while updating calculating plan specific values : %s", err), err, log)
+	}
+	operation.CloudProvider = string(provider.ProviderToCloudProvider(values.ProviderType))
+
 	if s.kimConfig.DryRun {
 		runtimeCR := &imv1.Runtime{}
-		err := s.updateRuntimeResourceObject(runtimeCR, operation, runtimeResourceName, kymaResourceName, kymaResourceNamespace)
+		err := s.updateRuntimeResourceObject(values, runtimeCR, operation, runtimeResourceName, operation.CloudProvider)
+
 		if err != nil {
 			return s.operationManager.OperationFailed(operation, fmt.Sprintf("while updating Runtime resource object: %s", err), err, log)
 		}
```

Comment:
Please check fmt.Println lines and remove them.

Reply:
What are you referring to?

Reply:
Discussed in private communication.